lib/AST/Bounds.cpp

```diff
@@ -972,34 +992,25 @@ class DeclBoundsFinder : public clang::DeclVisitor<DeclBoundsFinder>,
     VisitCommonDeclaratorDecl(decl);
   }
 
-  // TODO(pag): Handle parameters from the canonical decl being injected into
-  //            out-of-line methods on class templates.
-  void VisitParmVarDecl(clang::ParmVarDecl *decl) {
-    Expand(decl->getSourceRange());
-    if (!decl->hasInheritedDefaultArg() && decl->hasDefaultArg()) {
-      Expand(decl->getDefaultArgRange());
+  const ASTImpl::FunctionProto *FunctionProtoFor(clang::FunctionDecl *func) {
```

Comment:
Unclear name - I have no idea what it does.

Reply:
I have commented it.